### ISSUE TITLE
False positive on WORKDIR with quoted values

### ISSUE THREAD

**Issue by ferrarimarco** [hadolint/hadolint#261]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

The attached Dockerfile should be validated with no errors.

### Actual behavior

hadolint is reporting DL3000 on the attached Dockerfile because the `WORKDIR` value has been quoted. The use of an environment variable does not matter as I get the same error when using "/usr/src" directly in the WORKDIR directive.

The output also shows double quotes:

```
Linting ./Dockerfile
./Dockerfile:6 DL3000 Use absolute WORKDIR
ERROR: WORKDIR path ""/usr/src"" must be absolute on line 12

ERROR: ./Dockerfile failed.
```

### Steps to reproduce the behavior

1. `docker run -v "$(pwd)":/mnt --rm -w="/mnt" --name hadolint hadolint/hadolint hadolint Dockerfile`

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.11.2-0-g9c5e959
```

**Dockerfile** (if relevant)

```Dockerfile
FROM koalaman/shellcheck-alpine:v0.5.0

LABEL maintainer "Marco Ferrari <ferrari.marco@gmail.com>"

ENV SOURCE_DIRECTORY /usr/src
WORKDIR "$SOURCE_DIRECTORY"
```

**Additional environment details (OS, stack version, etc.)**
- Ubuntu 18.04
- Docker version 18.06.0-ce, build 0ffa825

Related issues: #28

**Comment by oliv3r** [comment]
Even when just wiriting `WORKDIR /usr/src` it still fails with the same error (without having it in a variable)

**Comment by bravecobra** [comment]
Same issue here  when using `WORKDIR "/src/services/ordering"`

**Comment by SimSonic** [comment]
Same issue with "/u/app"

**Comment by stefanpl** [comment]
@lukasmartinelli any chance of getting this fixed? Sadly, I don't know how to Haskell :)
